test_checker.cpp

```diff
@@ -0,0 +1,8 @@
+#include "checker.hpp"
+#include <assert.h>
+using namespace std;
+
+int main() {
+  assert(batteryIsOk(25, 70, 0.7) == true);
```

Comment:
Test cases need to be strengthened further. At least minimum set of permutation and combinations of all three parameters should be considered. For example one parameter will have 2 boundary values and 3 in between values, i.e. <, ==, > threshold values.

Having strong test cases is always helpful in maintaining code quality and guarding against accidental changes during extension.